test/test-debuglet.ts

```diff
@@ -377,6 +377,32 @@ describe('Debuglet', () => {
       assert.deepEqual(mergedConfig, compareConfig);
     });
 
+    it('should not start when workingDirectory is the a directory', (done) => {
```

Comment:
nit: typo ("the root directory"?)

Reply:
It is supposed to say "a root directory" because Windows can have multiple root directories.

Reply:
Fixed